docs/contributing.md

```diff
@@ -0,0 +1 @@
+../CONTRIBUTING.md
```

Comment:
The links on top of the contributing.md are not ordered by heading in the markdown

Reply:
done